source/includes/stackpath/_delivery_rules.md

```diff
@@ -49,6 +49,91 @@ Attributes | &nbsp;
 `stackId`<br/>*string* | The ID of the stack that the delivery rule belongs to.
 `siteId`<br/>*string* | The ID of the site that the delivery rule is applied to.
 
+<!-------------------- CREATE A DELIVERY RULE -------------------->
+
+### Create a delivery rule
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/deliveryrules?siteId=f9dea588-d7ab-4f42-b6e6-4b85f273f3db"
+```
+
+> Request body example for creating a delivery rule:
+
+```js
+{
+ "name": "rule",
+  "deliveryRuleConditions": [
+  {
+    "trigger": "HEADER",
+    "operator": "MATCHES",
+    "target": "my-header",
+    "httpMethods": ["GET"]
```

Comment:
I think its worth removing the `httpMethods` from the example since this would be ignored unless the `trigger` was `HTTP_METHOD`.

Reply:
wow 🤦🏻 Good point!

Reply:
Fixed in db21586ef5b052e4398519b44633184ed6ae92da